src/gui/AstroCalcDialog.hpp

```diff
@@ -521,19 +521,21 @@ private slots:
 	//! Iteration to calculate minimum distance from Besselian elements
 	double getJDofMinimumDistance(double JD);
 	//! Iteration to calculate JD of solar eclipse contacts
-	double getJDofContact(double JD, bool beginning, bool penumbral, bool external);
+	double getJDofContact(double JD, bool beginning, bool penumbral, bool external, bool outerContact);
 	//! Iteration to calculate contact times of solar eclipse
-	double getDeltaTofContact(double JD, bool beginning, bool penumbra, bool external);
+	double getDeltaTofContact(double JD, bool beginning, bool penumbra, bool external, bool outerContact);
```

Comment:
I did not see this before: I think the "deltaT" is actually a deltaTime (offset from central time)? Please rename to getDeltaTimeOfContact to avoid confusion around the "real" DeltaT (&Delta;T).

Reply:
Done.